src/env/from_envp_test.c

```diff
@@ -47,6 +47,29 @@ Test(env_from_envp, import_envp_containing_equal) {
 	cr_assert_str_eq(value, "ugh=1");
 }
 
+Test(env_from_envp, import_envp_no_equal) {
+	char		*arr[] = {"FOO=123", "BAR2345", "UGH=3", NULL};
+	t_env		env;
+	int			ret;
+	char		*value;
+
+	ft_bzero(&env, sizeof(t_env));
+	ret = env_from_envp(&env, arr);
+	cr_assert_eq(ret, 0);
+
+	value = env_get(&env, "FOO");
+	cr_assert_str_eq(value, "123");
+
+	value = env_get(&env, "UGH");
+	cr_assert_str_eq(value, "3");
+
+	value = env_get(&env, "BAR2345");
+	cr_assert_eq(value, NULL);
```

Comment:
Right now the warning is printed out to the terminal when running tests:

```
Warning: env: 'BAR2345', no '=' present
[====] Synthesis: Tested: 22 | Passing: 22 | Failing: 0 | Crashing: 0
```

I would recommend redirecting the stderr and testing it's value so it doesn't get printed:

```diff
diff --git a/src/env/from_envp_test.c b/src/env/from_envp_test.c
index 72597ca..61ae672 100644
--- a/src/env/from_envp_test.c
+++ b/src/env/from_envp_test.c
@@ -11,6 +11,7 @@
 /* ************************************************************************** */

 #include <criterion/criterion.h>
+#include <criterion/redirect.h>
 #include "private.h"

 Test(env_from_envp, import_envp) {
@@ -47,7 +48,7 @@ Test(env_from_envp, import_envp_containing_equal) {
     cr_assert_str_eq(value, "ugh=1");
 }

-Test(env_from_envp, import_envp_no_equal) {
+Test(env_from_envp, import_envp_no_equal, .init = cr_redirect_stderr) {
     char        *arr[] = {"FOO=123", "BAR2345", "UGH=3", NULL};
     t_env       env;
     int         ret;
@@ -68,6 +69,8 @@ Test(env_from_envp, import_envp_no_equal) {

     value = env_get(&env, "BAR");
     cr_assert_eq(value, NULL);
+
+    cr_assert_stderr_eq_str("Warning: env: 'BAR2345', no '=' present\n");
 }

 Test(env_from_envp, import_envp_null) {
```